website/index.html

```diff
@@ -1,102 +1,236 @@
+
+                <p class="info-text alert alert-secondary">
+                  Contract Status: <span id="contractStatus">Not clicked</span>
+                </p>
+              </div>
             </div>
           </div>
-        </div>
 
-        <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12 d-flex align-items-stretch">
-          <div class="card full-width">
-            <div class="card-body">
-              <h4 class="card-title">Send Eth</h4>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="sendButton" disabled>Send</button>
-              <hr />
-              <h4>Sign Typed Data</h4>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="signTypedData" disabled>Sign</button>
-              <p class="info-text alert alert-warning">Sign Typed Data Result: <span id="signTypedDataResult"></span></p>
+          <div
+            class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12 d-flex align-items-stretch"
+          >
+            <div class="card full-width">
+              <div class="card-body">
+                <h4 class="card-title">
+                  Send Eth
+                </h4>
+
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="sendButton"
+                  disabled
+                >
+                  Send
+                </button>
+
+                <hr />
+
+                <h4>
+                  Sign Typed Data
+                </h4>
+
+                <button
+                  class="btn btn-primary btn-lg btn-block mb-3"
+                  id="signTypedData"
+                  disabled
+                >
+                  Sign
+                </button>
+
+                <p class="info-text alert alert-warning">
+                  Sign Typed Data Result: <span id="signTypedDataResult"></span>
+                </p>
+              </div>
             </div>
           </div>
-        </div>
 
-        <div class="col-xl-4 col-lg-6 col-md-12 col-sm-12 col-12 d-flex align-items-stretch">
-          <div class="card full-width">
-            <div class="card-body">
-              <h4 class="card-title">Send Tokens</h4>
-              <p class="info-text alert alert-success">Token: <span id="tokenAddress"></span></p>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="createToken" disabled>Create Token</button>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="transferTokens" disabled>Transfer Tokens</button>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="approveTokens" disabled>Approve Tokens</button>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="transferTokensWithoutGas" disabled>Transfer Tokens
-                Without Gas</button>
-              <button class="btn btn-primary btn-lg btn-block mb-3" id="approveTokensWithoutGas" disabled>Approve Tokens
-                Without Gas</button>
```

Comment:
@Gudahtt while this appears as `Approve Tokens Without Gas`, in reality the value was something like `Approve Tokens                      Wihtout Gas`.

Reply:
Added this example to the README, too.